drivers/eks/eks_driver.go

```diff
@@ -173,6 +208,11 @@ func (d *Driver) GetDriverUpdateOptions(ctx context.Context) (*types.DriverFlags
 		Options: make(map[string]*types.Flag),
 	}
 
+	driverFlag.Options["kubernetes-version"] = &types.Flag{
```

Comment:
I think this should have a default string value of `1.10`. I think this is causing some problems (see below comments).